### PR TITLE
Lmp staging: fix the compilation lock of the expensive recipes 

### PR DESCRIPTION
**Pull request by quaresmajose**

### CODE REVIEW

File: meta-lmp-base/classes/lmp-staging.bbclass [hunk 1]
```diff
@@ -5,15 +5,15 @@
 #
 # Copyright 2022 (C) Foundries.IO LTD
```
Comment (MrCry0) [inline meta-lmp-base/classes/lmp-staging.bbclass:6]: We need to add 2023 here as well.

File: meta-lmp-base/classes/lmp-staging.bbclass [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
 
-INHERIT_KERNEL_MODSIGN = ""
+LMPSTAGING_INHERIT_KERNEL_MODSIGN = ""
```
Comment (MrCry0) [inline meta-lmp-base/classes/lmp-staging.bbclass:8]: Could you expand the commit message with an explanation of why we need these changes?

Comment (quaresmajose) [reply meta-lmp-base/classes/lmp-staging.bbclass:8]: The idea is to clarify that these variables are used in this classe. I will add it to the commit

File: meta-lmp-base/classes/lmp-staging.bbclass (hunk 1 above)
Comment (quaresmajose) [reply meta-lmp-base/classes/lmp-staging.bbclass:6]: I think is better to remove the year instead of have a requirement to update it every year.

**Review by MrCry0 — APPROVED** [review]
LGTM

File: meta-lmp-base/classes/lmp-staging.bbclass (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/classes/lmp-staging.bbclass:6]: Copyright is related to the year range that is valid for it, so we can have it 2022-2023 atm.

**Comment by ricardosalveti** [comment]
Small typo at the commit message from https://github.com/foundriesio/meta-lmp/pull/1027/commits/1befe767ef994f7dd360842603275ebc3eff0230

s/beackported/backported/g

File: meta-lmp-base/classes/lmp-staging.bbclass (hunk 1 above)
Comment (quaresmajose) [reply meta-lmp-base/classes/lmp-staging.bbclass]: updated the year, in 2024 will remove this part.

**Comment by quaresmajose** [comment]
> Small typo at the commit message from [1befe76](https://github.com/foundriesio/meta-lmp/commit/1befe767ef994f7dd360842603275ebc3eff0230)
> 
> s/beackported/backported/g

Also fixed

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**